app/uk/gov/hmrc/hmrcemailrenderer/templates/mandate/mandateAgentActivates.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, params("service").toString) {
+
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Your agent has now accepted your request, they now has the authority to act for you on ATED.</p>
```

Comment:
We have already replaced that with the service names being sent being passed and that's not an acronym. Would be pushing in the changes...